draft-ietf-quic-transport.md

```diff
@@ -3266,7 +3267,7 @@ sender or during delivery to the application at the receiver.
 
 When new data is to be sent on a stream, a sender MUST set the encapsulating
 STREAM frame's offset field to the stream offset of the first byte of this new
-data.  The first byte of data that is sent on a stream has the stream offset 0.
+data.  The first byte of data that is sent on a stream has an offset of 0.
```

Comment:
I think this should read `The first byte of data on a stream has an offset of 0.`

What you send first depends on what your receive and if you're a proxy you don't want to have to necessarily buffer data until you receive offset 0 just to forward it.